src/routes/committee/setlist/+page.ts

```diff
@@ -0,0 +1,76 @@
+import { db, verifyUserLoggedIn } from '$lib/firebase/client/firebase'
```

Comment:
Dit is allemaal pretty similar aan wat er voor schedule werd gedaan of niet. remind me om dit later ook aan te passen dan.